### PR TITLE
Send quality stats to prometheus.

### PR DESCRIPTION
**Pull request by boks1971**

### CODE REVIEW

File: pkg/telemetry/prometheus/quality.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
+package prometheus
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/livekit/protocol/livekit"
+)
+
+var (
+	qualityRating prometheus.Histogram
```
Comment (boks1971) [inline pkg/telemetry/prometheus/quality.go:10]: This is the three bar rating we have. So, won't make much sense to do p50, p90, p95 as there are only three bins (guess we can still do p90, p99. But, probably have to invert the ratings, i. e. if we expect `EXCELLENT` to have the highest number of entries, that is the last bin. Flipping it and making `EXCELLENT` the lowest bin might be more intuitive to think of p90.

But, I am think we should just display each bin as line graph (assuming that is possible) and just see how the three move across time.

File: pkg/telemetry/prometheus/quality.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+	qualityScore  prometheus.Histogram
```
Comment (boks1971) [inline pkg/telemetry/prometheus/quality.go:11]: This is the psuedo-MOS score. So, can do p50, p90, p95. Also has more granular bins.

File: pkg/telemetry/prometheus/quality.go [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+	qualityDrop   *prometheus.CounterVec
```
Comment (boks1971) [inline pkg/telemetry/prometheus/quality.go:12]: This tracks the number of tracks that have a drop in quality in the last evaluation window. Note the above two are recorded at participant level. A participant quality is the weakest quality of all its tracks (published and subscribed). This counts number of tracks that has dropped in quality in the last evaluation window and is a vector to record upstream and downstream quality drops independently.

**Review by davidzhao — APPROVED** [review]
looks good!

**Pull request merged**